lib/resources/r4/document_reference.yaml

```diff
@@ -212,6 +212,12 @@ fields:
    {
      "data": "PGh0bWw+Cjx0aXRsZT4gVGVzdCBEb2N1bWVudCA8L3RpdGxlPgoKRG9jdW1lbnQgY29udGVudCEKCjwvaHRtbD4="
    }
+  note: >
+    You can validate your document using any available strict XHTML 1.0 validator (eg: <a href="http://validator.w3.org/#validate_by_upload+with_options">w3 validator</a>
```

Comment:
Can you add something like: "If providing XHTML or HTML content, " to indicate that the validations and sanitization will only be run on those particular content types.